src/xdocs/config_filters.xml

```diff
@@ -477,8 +481,8 @@ public class UserService {
              </tr>
           </table>
           <p>
```

Comment:
This part is missing in the file `SuppressionFilter.java`.
I think the `properties` section should contain only the properties table

Reply:
I agree, I couldn't find it but there is an issue somewhere about creating a new section in the check/filter for external configuration files like this.